src/inc/corprof.idl

```diff
@@ -3983,3 +3983,27 @@ interface ICorProfilerAssemblyReferenceProvider : IUnknown
     // assembly specified in the wszAssemblyPath argument of the GetAssemblyReferences callback.
     HRESULT AddAssemblyReference(const COR_PRF_ASSEMBLY_REFERENCE_INFO * pAssemblyRefInfo);
 };
+
+
+/***************************************************************************************
+** ICLRProfiling                                                                     **
+** Activated using mscoree!CLRCreateInstance. Export AttachProfiler API to profilers **
+***************************************************************************************/
+cpp_quote("#ifndef DACCESS_COMPILE")
```

Comment:
you can get rid of this #ifdef we added, I'm pretty sure the metahost change that eliminated the extern_guid(...) is what fixed the build issue.